### PR TITLE
Systems Context

### PR DESCRIPTION
**Pull request by zacharyblasczyk**

### CODE REVIEW

File: apps/webservice/next.config.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const config = {
   },
 
   experimental: {
-    instrumentationHook: true,
+    instrumentationHook: false,
```
Comment (jsbroks) [inline apps/webservice/next.config.js:28]: ?

File: apps/webservice/tsconfig.json [hunk 2]
```diff
@@ -11,6 +11,10 @@
     "tsBuildInfoFile": "node_modules/.cache/tsbuildinfo.json",
     "module": "esnext"
   },
-  "include": [".", ".next/types/**/*.ts"],
+  "include": [
+    ".",
+    ".next/types/**/*.ts",
+    "src/app/[workspaceSlug]/(targets)/target-providers/.tsx"
```
Comment (jsbroks) [inline apps/webservice/tsconfig.json:17]: ?

**Comment by jsbroks** [comment]
could put it under deployments section
can you reuse the current status icons stuff we do in the jobs tables
and the top right deployment count doesn't follow the same styling as our targets count table view
I would also remove the table header text

**Comment by zacharyblasczyk** [comment]
Closing for now.

**Pull request closed without merging**